java/ql/src/semmle/code/java/frameworks/apache/Lang.qll

```diff
@@ -2,6 +2,7 @@
 
 import java
 private import semmle.code.java.dataflow.FlowSteps
+private import semmle.code.java.dataflow.ExternalFlow
```

Comment:
We need bidirectional import. Currently that happens indirectly, but I think it's best to import this file directly in the `private module Frameworks` in `ExternalFlow.qll`.

Reply:
Done